integration_tests/tests/vault/create_token_metadata.rs

```diff
@@ -0,0 +1,60 @@
+use solana_program::pubkey::Pubkey;
+
+use crate::fixtures::{fixture::TestBuilder, vault_client::VaultRoot};
+
```

Comment:
add test for bad vault admin signer

Reply:
I added a new test case `test_wrong_admin_signed`